### ISSUE TITLE
Clipboard could not get data

### ISSUE THREAD

**Issue by akallabeth** [FreeRDP/FreeRDP#4826]
* Fedora 28
* Nightly builds 
* Command line `/opt/freerdp-nightly/bin/xfreerdp /v:hostname /network:auto /gfx:avc444 /multimon /sound /u:user /p:pwd`

```[13:07:38:595] [22331:22343] [ERROR][com.freerdp.client.x11] - failed to get clipboard data in format UTF8_STRING [source format CF_UNICODETEXT]
[13:07:38:595] [22331:22343] [ERROR][com.freerdp.channels.cliprdr.client] - ServerFormatDataResponse failed with error 1359!
[13:07:38:595] [22331:22343] [ERROR][com.freerdp.channels.cliprdr.client] - cliprdr_process_format_data_response failed with error 1359!
[13:07:38:595] [22331:22343] [ERROR][com.freerdp.channels.cliprdr.client] - cliprdr_order_recv failed with error 1359!
[13:07:38:595] [22331:22332] [ERROR][com.freerdp.core] - cliprdr_virtual_channel_client_thread reported an error. Error was 1359
[13:07:38:595] [22331:22332] [ERROR][com.freerdp.core] - checkChannelErrorEvent() failed - 0
[13:07:38:595] [22331:22332] [INFO][com.freerdp.client.x11] - Network disconnect!
[13:07:38:596] [22331:22332] [ERROR][com.freerdp.client.x11] - Failed to check FreeRDP file descriptor
```

**Comment by gnshkl1** [comment]
@akallabeth i am also facing same session disconnect  issue with 3rd party channel ,when clipboard is enabled

**Comment by dollkillla** [comment]
cliprdr_virtual_channel_client_thread reported an error. Error was 1359
4.17.0-3-amd64 #1 SMP Debian 4.17.17-1 (2018-08-18) x86_64 GNU/Linux
trying with command 'wlfreerdp /u:xx  /v:x.x.x.x:x'

**Comment by akallabeth** [comment]
@dollkillla you're victim of #3414 ;)

**Comment by akallabeth** [comment]
Should be fixed with #4879 merged.
Reopen / Create a new issue if still not solved.
